### DISCUSSION TITLE
Is L (for last day of month) supported in crons?

### DISCUSSION THREAD

**Discussion by matthieuh** [quirrel-dev/quirrel#201] [Q&A] [answered]
As describe in https://github.com/harrisiirak/cron-parser doc `L` for last day of month seems supported (cf PR: https://github.com/harrisiirak/cron-parser/pull/183).

Is this syntax supported in Quirrel?

Seems not working to me. I both tried `* * * L * *` and `* * L * *`.

**Comment by Skn0tt** [comment, marked as answer]
There was a regex in-place that didn't support all features of `cron-parser`. I put up https://github.com/quirrel-dev/quirrel/pull/208, which will solve this problem.

**Reply by Skn0tt** [reply]
Published in v1.3.3 ☺️

**Reply by matthieuh** [reply]
Awesome thank you!
